template-backend-extension/src/widget/WidgetGlobalSettings.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo } from 'react';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+
+import { Link } from '@lumx/react';
+import { NotificationsProvider, PredefinedErrorBoundary, useLanguage } from 'lumapps-sdk-js';
+
+import messagesEn from '../translations/en.json';
+import messagesFr from '../translations/fr.json';
+
+type WidgetGlobalSettings = import('lumapps-sdk-js').GlobalSettingsComponent<any>;
+
+/**
+ * Render the widget global settings component.
+ *
+ * @param {Object} props The settings component properties.
+ */
+const IntlWidgetGlobalSettings: WidgetGlobalSettings = ({ properties = {}, exportProp }) => {
+    const contactMail = 'contact@mail.com';
+    const subjectMail = 'Extension activation';
+
+    return (
+        <div>
+            <p className="lumx-typography-body2">
```

Comment:
LGTM.
Minor issue, it seems there is some prettier issues in this file. Except that, seems good to me

Reply:
Tests OK : 
Could "install" the backend template : OK
Everything load correctly : OK
Only access to Global settings : OK